cypress.config.js

```diff
@@ -27,6 +27,15 @@ module.exports = defineConfig({
             tag: faker.lorem.word()
           };
         },
+        generateNewData() {
+          const randomNumber = Math.ceil(Math.random(1000) * 1000);
+          return {
+            username: faker.name.firstName() + `${randomNumber}`,
+            bio: faker.lorem.words(),
+            email: 'user' + `${randomNumber}` + '@gmail.com',
```

Comment:
The email address should be generated using faker to ensure uniqueness and format correctness. Instead of 'user' + `${randomNumber}` + '@gmail.com', use 'faker.internet.email()'.